python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -744,6 +748,7 @@ def dispose(self):
         self._compute_log_manager.dispose()
 
     # run storage
+    @public
```

Comment:
is there a run record variant of this we would like to promote instead? I dunno, the cost on this one is very low and pretty high utility so im fine if we end up supporting both